README.md

```diff
@@ -52,6 +87,24 @@ variables:
 * `LEATHERMAN_<LIBRARY>_LIBS` The contents of both
   `LEATHERMAN_<LIBRARY>_LIB` and `LEATHERMAN_<LIBRARY>_DEPS`
 
+### CMake Helpers Provided by Leatherman
+
+In addition to the C++ library components, Leatherman provides a few
+CMake helpers. These will be automatically added to your
+`CMAKE_MODULE_PATH` when `find_package` is processed.
```

Comment:
I'm kind of tempted to just include `options`, `cflags`, and `leatherman` in the Finder and obviate the need to explicitly include them. I can't think of a situation where we *wouldn't* want them to be there.

My only concern with that is certain things aren't well-namespaced. I may hold off on that until a separate PR where I can rename all the macros to have a `leatherman_` or `lth_` prefix.